### ISSUE TITLE
product price does contain HTML entities like €&nbsp;18,00€&nbsp;20,00(NaN%) OFF

### ISSUE THREAD

**Issue by pimftw** [imranhsayed/woo-next#114]
When I clone a fresh copy of this repo and install the added plguins it results in the product price containing HTML special characters.
![Schermafbeelding 2022-05-30 om 10 58 53](https://user-images.githubusercontent.com/1047830/170956758-5cfc6e38-da2a-43e1-8b81-12528019e1f6.png)

**Comment by w3bdesign** [comment]
@pimftw 

How does the price output look like in your Woocommerce frontend? 
Do the prices display correctly there?

**Comment by PaulCordo** [comment]
Yes might be due to this update from wp-graphql-woocommerce https://github.com/wp-graphql/wp-graphql-woocommerce/issues/75
You now need to add a filter next to price values in your request like `price(format: RAW)`
